triggerReleasePipeline.sh

```diff
@@ -27,6 +27,7 @@ set -euo pipefail
 JDKVERSION="$1"
 ADOPTIUM_REPO=${2:-"https://github.com/adoptium/$JDKVERSION.git"}
 BRANCH=${3:-"master"}
+SOLARIS=${4:-"normal"}
```

Comment:
Possibly want to update line 25 which mentions the parameters to include this one.